jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <j-tray-plugin
+    :description="docs_description"
+    :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#cubeviz-sonify-data'"
+    :uses_active_status="uses_active_status"
+    @plugin-ping="plugin_ping($event)"
+    :keep_active.sync="keep_active"
+    :popout_button="popout_button"
+    :scroll_to.sync="scroll_to">
+
+    <j-plugin-section-header>Sonify Cube</j-plugin-section-header>
+    <v-alert v-if="!has_strauss" type="warning" style="margin-left: -12px; margin-right: -12px">
+      To use Sonify Data, install strauss and restart Jdaviz. You can do this by running `pip install .[strauss]`
+      in the command line and then launching Jdaviz.
+    </v-alert>
+    <v-row>
+      <j-docs-link>Choose the input cube (currently hardcoded to be the cube in the flux viewer) and spectral subset.</j-docs-link>
```

Comment:
is this redundant with the hints in the subset dropdown?  If there is concern that the input cube isn't clear, we could add a dropdown with a single entry that matches the assumed one.